velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -1555,6 +1589,7 @@ TEST_F(PrestoSerializerTest, deserializeSingleColumn) {
       DOUBLE(),
       VARCHAR(),
       TIMESTAMP(),
+      OPAQUE<Foo>(),
```

Comment:
This is just adding a type to `deserializeSingleColumn` which is not used for serialization across the wire (See method comment of `deserializeSingleColumn` for more info). Therefore, you would also want to add test coverage for the serialize (both batch and iterative) and deserialize methods.

Also would have to add tests for `estimateSerializedSize` when you implement it